test/tests.sh

```diff
@@ -40,7 +40,7 @@ WD_PATH=${PWD}/..
 
 wd()
 {
-    "${WD_PATH}"/wd.sh "$@"
+    source "${WD_PATH}"/wd.sh "$@"
```

Comment:
🏷️ Including just this change - nothing else to tests or `wd.sh` - will cause `test_config` to fail with:

```
test_config
ASSERT:expected:<0> but was:<1>
rm: cannot remove '/tmp/tmp.Qj3BlEu99M': No such file or directory
```

Reply:
Also want to add that the default `WD_CONFIG` in tests seems to not affect the "outside" variable, so it's not changed!